docs/pages/management/admin/trustedclusters.mdx

```diff
@@ -1,500 +1,594 @@
+
+In this example, the leaf cluster `us-east-1a` denies Alice access to the `mongodb.prod.us-east-1a` resource
+because her `full-access` role in the root cluster is mapped to the `stage-access` role in this leaf cluster.
+With role mapping, leaf cluster administrators can control the privileges external users are granted.
+Although role mapping can be as simple as assigning users to the same roles in the root and leaf clusters,
+you can also use mapping to downgrade a user's privileges or restrict access to specific resources.
+
+Now that you know what a trusted cluster is and how it works, you can use this guide 
+to learn how to:
+
+- Identify root and leaf clusters.
+- Add trusted cluster resources.
+- Generate an invitation token to establish a trust relationship between a root and leaf cluster.
 - Establish permission mapping between clusters using Teleport roles.
+- Enable and disable trust between clusters.
 
 ## Prerequisites
 
-<Tabs>
-<TabItem scope="team" label="Teleport Team">
-
-- A Teleport Team account. If you do not have one, visit the [signup
-  page](https://goteleport.com/signup/) to begin your free trial.
-
-- A second Teleport cluster, which will act as the leaf cluster. For details on
-  how to set up this cluster, see our [Getting Started](../../index.mdx)
-  guide.
-
-  As an alternative, you can set up a second Teleport Team account.
-
-- (!docs/pages/includes/cloud/tctl-tsh-prerequisite.mdx!)
-
-- A Teleport Node that is joined to one of your clusters. We will refer to this
-  cluster as the **leaf cluster** throughout this guide.
-
-  See [Join Services to your Cluster](../../agents/join-services-to-your-cluster.mdx) for
-  how to launch a Teleport Node in your cluster.
-
-</TabItem>
-<TabItem scope={["oss"]} label="Open Source">
-
-- Two running Teleport clusters. For details on how to set up your clusters, see
-  our [Getting Started](../../index.mdx) guide.
-
-- (!docs/pages/includes/tctl-tsh-prerequisite.mdx!)
-
-  See [Installation](../../installation.mdx) for details.
-
-- A Teleport agent that is joined to one of your clusters. We will refer to this
-  cluster as the **leaf cluster** throughout this guide.
-
-  See [Join Services to your
-  Cluster](../../agents/join-services-to-your-cluster.mdx) for how to launch a
-  Teleport agent in your cluster.
+To complete the steps in this guide, verify your environment meets the following requirements:
 
-</TabItem>
-<TabItem
-  scope={["enterprise"]} label="Enterprise">
+- Access to **two** Teleport cluster instances in your environment:
 
-- Two running Teleport clusters. For details on how to set up your clusters, see
-  our Enterprise [Getting
-  Started](../../choose-an-edition/teleport-enterprise/introduction.mdx) guide.
+  - Teleport Community Edition or Teleport Enterprise if you manage the Teleport clusters on your own network.
+  - Teleport Team or Teleport Enterprise Cloud if you access Teleport as a managed cloud-based service.
```

Comment:
We might also want to mention compatibility. The leaf cluster should be at the same version or at most 1 major version behind the version of the root.